src/pyedb/dotnet/edb.py

```diff
@@ -1523,12 +1523,8 @@ def import_gds_file(
             ``True`` when successful, ``False`` when failed.
 
         """
-        if tech_file or map_file:
-            control_file_temp = os.path.join(tempfile.gettempdir(), os.path.split(inputGDS)[-1][:-3] + "xml")
-            ControlFile(xml_input=control_file, tecnhology=tech_file, layer_map=map_file).write_xml(control_file_temp)
-        elif tech_file:
-            self.logger.error("Technology files are supported only in Linux. Use control file instead.")
```

Comment:
Can technology file be used with linux now ?

Reply:
I dont think so, but why to return False if tech_file is used? Maybe we should add it as a warning?

Reply:
Then we should probably refactor to leverage `is_linux` and trigger the error message if `tech_file and not is_linux`. Otherwise, users won't know that technology files are only supported in linux.
If not then we can proceed with your proposed changes.